.github/workflows/codeql-analysis.yml

```
@@ -69,4 +69,4 @@
     #    make release
 
     - name: Perform CodeQL Analysis
-      uses: github/codeql-action/analyze@1b549b9259bda1cb5ddde3b41741a82a2d15a841 # v3.28.13
+      uses: github/codeql-action/analyze@45775bd8235c68ba998cffa5171334d58593da47 # v3.28.15
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)